server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -77,6 +77,8 @@ public class DiscoveryNode implements VerifiableWriteable, ToXContentFragment {
 
     static final String COORDINATING_ONLY = "coordinating_only";
 
+    static final String SEARCH_ONLY = "searchonly";
```

Comment:
In the event we are not able to repurpose the search role lets make this consistent with the coordinating_only, - `search_only`